servers/rendering/renderer_rd/effects/debug_effects.cpp

```diff
@@ -51,6 +51,17 @@ DebugEffects::DebugEffects() {
 		raster_state.wireframe = true;
 		shadow_frustum.pipelines[SFP_WIREFRAME].setup(shadow_frustum.shader.version_get_shader(shadow_frustum.shader_version, 0), RD::RENDER_PRIMITIVE_LINES, raster_state, RD::PipelineMultisampleState(), RD::PipelineDepthStencilState(), RD::PipelineColorBlendState::create_disabled(), 0);
 	}
+
+	{
+		// Combined shadow map debug shader
```

Comment:
```suggestion
		// Combined shadow map debug shader.
```
Code style nitpick, same elsewhere (with first letter capitalized in some cases)